CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 -----
 
+## 0.2.9 (2018-08-10)
+
+#### Fixes
+- Fixed error when changing episode status from episode status management ([#4783](https://github.com/pymedusa/Medusa/pull/4783))
+- Fixed multi-episode snatches not being marked as snatched in history ([#229](https://github.com/pymedusa/Medusa/issues/229))
+- Fixed whole seasons being downloaded as multi-episode replacement ([#4750](https://github.com/pymedusa/Medusa/issues/4750))
```

Comment:
Please merge develop and put these in the "Unreleased" section.
If we start setting version numbers and dates too soon we'll have to keep updating them until we actually release.

Reply:
Also, thank you for including #4783 which I forgot to add!